### ISSUE TITLE
Textarea not properly styled when focused

### ISSUE THREAD

**Issue by WebFactoryLtd** [creativetimofficial/material-dashboard-pro#9]
The problem is visible in this template but in other material ones as well. Focused textareas don't have the bottom border coloured like other input fields. For instance, [users page](http://demos.creative-tim.com/material-dashboard-pro/examples/pages/user.html) All input fields get a border-bottom on :focus, textarea doesn't.

**Comment by alexandru-paduraru** [comment]
@WebFactoryLtd sorry for the late response. We've checked and everything is working fine on our side. 

Here is what we see:

<img width="1047" alt="screen shot 2017-02-07 at 20 23 43" src="https://cloud.githubusercontent.com/assets/2517758/22705277/9633cd74-ed73-11e6-88fe-698d0560357f.png">

Everything working fine also with the animation.

Can you please give us more details about what OS, Browser and Device you use? 

Best,
Alex

**Comment by WebFactoryLtd** [comment]
Win10, Chrome v56 64bit
Placeholder changes size and moves up, and everything looks normal but the border-bottom stays the same regardless of the element being in focus or not. On inputs it works as expected. When I downloaded files and checked the same pages it worked on textareas. 

Tried FIrefox now. Behaves exactly like Chrome - border-bottom stays unchanged on focus. 

Found the culprit!!! Grammarly addon! Disabled it - works fine! Thank you for your time.
